tests/unit/protobuf/test_protobuf_compatibility.py

```diff
@@ -300,3 +300,37 @@ def test_compatibility_ordering_change2():
     assert result.is_compatible()
     assert len(result.result) == 1
     assert result.result[0].modification == Modification.FIELD_ADD
+
+
+def test_compatibility_field_tag_change():
+    self_schema = """\
+syntax = "proto3";
+package pkg;
+message Foo {
+    string fieldA = 1;
+    string fieldB = 2;
+    string fieldC = 3;
+    string fieldX = 4;
+}
```

Comment:
maybe add `string fieldD = 5;` since we are interested in the tag change, not in the field removal :)

Reply:
I just took over the example from the ticket. But true it's kinda confusing. I will just remove fieldD